recipes/dydx-v3-python/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "dydx-v3-python" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 667f5f501afdae2b328b95308882978dc6edd4e538f9873102317b3db31719ab
+  patches:
+    - relaxed_web3_upgrade.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0
+    - pip
+  run:
+    - python >=3.8.0
+    - aiohttp
+    - cytoolz
+    - dateparser
+    - ecdsa
+    - eth-keys
+    - eth-account
+    - mpmath
+    - requests
+    - sympy
+    - web3
+
+test:
```

Comment:
Can you reduce the test complexity and maybe only add an import? It seems like this is pure python, so you don't need to re-run the whole test.

Reply:
I understand, however, is it a big problem to keep them around?

Reply:
It makes it easier and less error prone so it would be better.

Reply:
It was addressed in the last push, now it is just `pip check`